spec/factories/publication.rb

```diff
@@ -36,7 +36,7 @@
 
     trait :other_work do
       publication_type {
-        Publication.publication_types.reject { |type| Publication.oa_publication_types.include?(type) }.sample
+        Publication.publication_types.reject { |type| Publication.journal_types.include?(type) }.sample
```

Comment:
.oa_publication_types was throwing occasionally throwing a false error. "Trade Journal Article" is sometimes selected as the publication type. After merge, it's changed to just "Journal Article" (which is correct) but is then causing the test to fail because it expected it not to change. I don't think the test should be adjusted because any other "Other Work" is expected to stay the same, so maybe reject .oa_publication_types and .journal_types? They are very similar with the only difference being that oa_publication_types includes conference proceedings and journal_types includes trade journal. Please let me know what you think.

Reply:
I think I fixed the test that was failing in a PR I merged this morning, so if you merge main in the tests should be good:
https://github.com/psu-libraries/researcher-metadata/commit/91f691c4189c7d7098722a99f109df008baf5642
The reason that test was failing was because it was still testing with the assumption that we hadn't changed the logic around OA publications a few weeks ago.  I may be talking about a different test than you, though?

Oa_publication_types should be all types that are subject to OA and the "Other Works" should contain works that are not subject to OA.  So for the 'other_work' trait, we would want to reject the OA publications (including Conference Proceedings) and not reject the others (including Trade Journal Article).